### PR TITLE
Default trusted forwarders for zksync

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

**Review by joaquim-verges — COMMENTED** [review]
Does OZ defender support zksync? Wonder if there's any relayer that works rn 😅 also with their AA setup, it might work a bit differently no?

Did we have a report of this being an issue already?

**Comment by kumaryash90** [comment]
@joaquim-verges yeah they had enabled zksync some time ago: https://forum.openzeppelin.com/t/defender-release-support-for-zksync-era/35565

Though we haven't yet received any reports related to gasless tx issues, but adding this just in case.
Will look at ZKSync AA, and if gasless is needed at all.

**Comment by joaquim-verges** [comment]
Nice, good catch anticipating this. Ship it

File: components/contract-components/hooks.ts [hunk 1]
```diff
@@ -543,24 +544,30 @@ export function useCustomContractDeployMutation(
...
+                chainId,
+                compilerMetadata.data?.name,
+              )
+            : await getTrustedForwarders(
+                sdk.getProvider(),
+                sdk.storage,
+                compilerMetadata.data?.name,
+              );
+
+          console.log("trusted forwarders: ", trustedForwarders);
```
Comment (nachoiacovino) [inline components/contract-components/hooks.ts:565]:
```suggestion
```

**Pull request merged**